src/main/webapp/style.css

```diff
@@ -0,0 +1,273 @@
+body {
+background-color: #ECECEC;
+}
+
+#content {
+  margin: auto;
+  max-width: 100%;
+}
+
+div.header {
+  background: #856C8B;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  box-sizing: border-box;
+  height: 100px;
+  left: 0px;
+  top: 0px;
+  width: 100%;
+}
+
+div.header h1 {
+  color: #FFFFFF;
+  font-family: Ruluko;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: normal;
+  height: 89px;
+  line-height: 69px;
+  margin-left: 20px;
+  margin-top: 15px;
+}
+
+input {
+  display: block;  
+  height: 25px;
+  margin-bottom: 8px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 1035px;
+}
+
+input:hover {
+  opacity: 0.7;
+}
+
+textarea {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  display: block;  
+  height: 70px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 1035px; 
+}
+
+textarea:hover {
+  opacity: 0.7;
+}
+
+#data-holder {
+  color: black;
+  font-size: 25px;
+  margin-bottom: 50px;
+  margin-top: 40px;
+}
+
+#comments-container {
+  background: #FFFFFF;
+  border: 0.25px solid black;  
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  height: 379px;
+  margin: auto;
+  width: 1035px;
+}
+
+#comments-header {
+  color: #856C8B; 
+  font-family: Aksara Bali Galang;
+  font-size: 30px;
+  font-style: normal;
+  font-weight: normal;
+  height: 58.54px;
+  line-height: 86px; 
+  margin: auto;
+  margin-bottom: 15px;
+  text-align: center;
+  width: 1035px;
+}
+
+button {
```

Comment:
How would I start that?

Reply:
It's just about moving around these blocks. Move all of the selectors that are tag names rather than classes or ids to the top of the file. You'd have:

body {
...
}

p {...}

a {...}

button {...}

input {...}

input:hover {...}

textarea {...}

etc.